examples/experimental/net-user-v2.yaml

```diff
@@ -6,6 +6,11 @@ images:
 - location: "https://cloud-images.ubuntu.com/releases/22.04/release/ubuntu-22.04-server-cloudimg-arm64.img"
   arch: "aarch64"
 
+hostResolver:
+  # Marking enabled false will make use of system dns / inbuilt dns resolver of usernet for user-v2 network
```

Comment:
Could you explain why it cannot be enabled?

Reply:
No, i was using this hostResolver.enabled: false as a toggle to enable the inbuilt DNS, Based on above decision i will refactor this accordingly